client/client.go

```diff
@@ -8,14 +8,14 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"log"
 	"net"
 	"strings"
 	"sync"
 	"sync/atomic"
 	"time"
 	"unsafe"
 
-	"github.com/appscode/go/log"
```

Comment:
@ssmccoy let me know if you'd like this removed. I noticed that this import was already there since  originally being introduced in appscode fork.